component-versions.md

```diff
@@ -2,6 +2,60 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+[Details=1.11.1]
+
+### Charms
+
+| Name | Revision |
+|----------|--------------|
+| cs:~anbox-charmers/lxd   |  ? |
+| cs:~anbox-charmers/ams    |  ? |
+| cs:~anbox-charmers/ams-node-controller    | ?  |
+| cs:~anbox-charmers/aar    |  ? |
+| cs:~anbox-charmers/anbox-stream-gateway   |  ? |
+| cs:~anbox-charmers/anbox-stream-agent  |  ?  |
+| cs:~anbox-charmers/coturn  | ? |
+| cs:~anbox-charmers/nats  |  ? |
+| cs:~anbox-charmers/ams-load-balancer | ? |
+| cs:~anbox-charmers/anbox-cloud-dashboard | ? |
+
+### Validated revisions of 3rd party charms
+
+| Name | Revision |
+|----------|--------------|
+| cs:~containers/easyrsa | 384 |
+| cs:~containers/etcd | 594 |
+
+### Bundles
+
+| Name | Revision |
+|----------|--------------|
+| cs:~anbox-charmers/anbox-cloud | ? |
+| cs:~anbox-charmers/anbox-cloud-core | ? |
+
+### Snaps
+
+| Name | Version |
+|----------|--------------|
+| ams    | 1.11.1 |
+| aar    | 1.11.1 |
+| ams-node-controller    | 1.11.1  |
+
+## Debian Packages
```

Comment:
Is this supposed to be an h2 heading (with h3's above)?

Reply:
Ah no, let me correct that.

Reply:
Done